### PR TITLE
I18n setup [story id: 171148785]

### PR DESCRIPTION
**Pull request by akanshak984**
This pull request cover 
1. databse.yml file added
2. internationalization 
3. gitignore updated 
4. path for credentials added in respective enviornment and
multi-enviornment credentials also added

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+#min_messages: notice
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+ <<: *default
+ database: <%= ENV['CAB_TEST'] %>
+ username: <%= ENV['DB_USERNAME'] %>
+ password: <%= ENV['DB_PASSWORD'] %>
```
Comment (smriti154) [inline config/database.yml]: @akanshak984 Don't we add staging and production configuration?

File: config/initializers/locale.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Where the I18n library should search for translation files
+I18n.load_path += Dir[Rails.root.join('lib', 'locale', '*.{rb,yml}')]
+# Permitted locales available for the application
+I18n.available_locales = [:en, :hi]
+# Set default locale to something other than :en
+I18n.default_locale = :en
```
Comment (smriti154) [inline config/initializers/locale.rb]: Please Run rubocop on this file.

File: config/locales/hi.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+hi:
+  hello: "नमस्कार"
```
Comment (smriti154) [inline config/locales/hi.yml]: This file can be added, when we have to provide multilingual support

File: config/database.yml (hunk 1 above)
Comment (akanshak984) [reply config/database.yml]: Yeah, I will update it!

File: config/initializers/locale.rb (hunk 2 above)
Comment (akanshak984) [reply config/initializers/locale.rb]: Okay! I will do that.

File: config/locales/hi.yml (hunk 3 above)
Comment (akanshak984) [reply config/locales/hi.yml]: So, should I add it or not?

**Pull request closed without merging**